app/views/dashboard/_pending_applications.haml

```diff
@@ -10,5 +10,6 @@
               %li= link_to record.user.proper_name, record
           - else
             No pending applications
-        = link_to 'View saved applications for ' + position.name,
+        - count = position.saved_applications.count
+        = link_to "View saved applications for #{position.name} (#{count})",
```

Comment:
I maintain that this should be wrapped in a conditional checking for none. If we're worried about UI elements being inconsistent, perhaps some text (`No saved applications`) in the event where there are 0?